docs/shopify_app/engine.md

```diff
@@ -15,7 +15,7 @@ While you can customize the login view by creating a `/app/views/shopify_app/ses
 
 ```ruby
 ShopifyApp.configure do |config|
-  config.login_url = 'https://my.domain.com/nested/login'
+  config.login_url = 'https://my.domain.invalid/nested/login'
```

Comment:
There are a specially reserved domains for this purpose: https://en.wikipedia.org/wiki/Example.com

Reply:
TIl. Changed it to `example.com`